lib/SIL/Utils/FieldSensitivePrunedLiveness.cpp

```diff
@@ -350,6 +355,10 @@ void TypeTreeLeafTypeRange::constructFilteredProjections(
       callback(newValue, TypeTreeLeafTypeRange(start, next));
       start = next;
     }
+    if (type.isValueTypeWithDeinit()) {
```

Comment:
I am fine with this as is... but I would like a larger comment here that doesn't just say 'self' has its own liveness. I wish it had the larger explanation or a reference back to the earlier explanation.

Reply:
Good point. although I need to merge this now to put up a 5.9 PR